Face/FaceSample.sln

```diff
@@ -3,7 +3,9 @@ Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio 15
 VisualStudioVersion = 15.0.27703.2042
 MinimumVisualStudioVersion = 10.0.40219.1
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "VerifyFaceToFace", "VerifyFaceToFace\VerifyFaceToFace.csproj", "{9090CBD6-25BE-4C58-A747-814D2E566956}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "VerifyFaceToFace", "VerifyFaceToFace\VerifyFaceToFace.csproj", "{9090CBD6-25BE-4C58-A747-814D2E566956}"
```

Comment:
Let's use `FAE04EC0-301F-11D3-BF4B-00C04F79EFBC` for both `Verify` and `Detect` Projects.